test/Conversion/tritongpu_to_llvm.mlir

```diff
@@ -1,4 +1,4 @@
-// RUN: triton-opt %s -split-input-file --convert-triton-gpu-to-llvm | FileCheck --check-prefixes=CHECK,GCN %s
+// RUN: (triton-opt %s -split-input-file --convert-triton-gpu-to-llvm --mlir-pass-pipeline-crash-reproducer=%t 2>/dev/null; true) | FileCheck --check-prefixes=CHECK,GCN %s
```

Comment:
`--mlir-pass-pipeline-crash-reproducer` option is essential here.
With this option separate tests cases are run in dedicated process.

Crashed test produce no output, which is useful for unsupported functionality for AMD compilation.
For example:
- not supported yet stuff, like in `convert_dot`.
- PTX specific stuff, like in `basic_async_wait`